.github/workflows/ci_cd.yml

```
@@ -1,5 +1,10 @@
 name: CI / CD
-on: push
+on:
+  push:
+  pull_request:
+    types: [opened, reopened]
+concurrency:
+  group: ${{ github.ref }}
```

issue(blocking): Per discussion with @freaz we concluded that this isn't correct approach. As far as we understand it, this will run workflow on PR only when it's opened, but not on consecutive pushes from forked repository.
Here's a suggested fix (pending internal announcement):

```suggestion
on:
  pull_request:
  push:
    branches:
      - main
```

---

Just to make sure, does this trigger when a PR is "updated" by pushing new commits to the source branch?

I don't think that's technically an action *on the pull request*, and [there's no activity type](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request) that obviously stands out to me as indicating this happened; which would make sense, as that has nothing to do with the PR itself, because it's a *push* to the source branch. But that would get filtered out, because it's not a push to `main`.

Am I making sense? What happens in that case?